### PR TITLE
Rename rustc-guide -> rustc-dev-guide

### PR DESCRIPTION
**Pull request by mark-i-m**
cc @rust-lang/wg-learning 

To be merged on monday after we rename the repo...

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Doesn't seem to like the `@` in your author field

File: posts/inside-rust/2020-03-09-rename-rustc-guide.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+author: mark-i-m
+description: "the guide has been renamed"
+team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
+---
+
+You may or may not be aware of two similarly named resources:
+- [The rustc book](https://doc.rust-lang.org/rustc/index.html)
+- [The rustc-guide](https://rust-lang.github.io/rustc-dev-guide/)
+
+What's difference? The "rustc book" is intended as a guide for _users_ of the
```
Comment (LeSeulArtichaut) [inline posts/inside-rust/2020-03-09-rename-rustc-guide.md]: Typo?
```suggestion
What's the difference? The "rustc book" is intended as a guide for _users_ of the

File: posts/inside-rust/2020-03-09-rename-rustc-guide.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+compiler. The "rustc-guide" is intended as a guide for _developers_ of the
+compiler. However, the names have always been confusingly similar.
+
+**For that reason, we have decided to rename the `rustc-guide` to
+`rustc-dev-guide`.**
+
+We have put significant work into finding and updating links around the
+`rust-lang/*` repos. We have also put up a website in place of the former
+`rustc-guide` website that points to the book and rustc-dev-guide.
```
Comment (LeSeulArtichaut) [inline posts/inside-rust/2020-03-09-rename-rustc-guide.md]: I thought this website was a redirection point to the rustc-dev-guide... Maybe clarify this with the WG?

Comment (mark-i-m) [reply posts/inside-rust/2020-03-09-rename-rustc-guide.md]: Woops yes, you're right. I meant to ask on zulip, but then I got some tracked and forgot

File: posts/inside-rust/2020-03-09-rename-rustc-guide.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (LeSeulArtichaut) [inline posts/inside-rust/2020-03-09-rename-rustc-guide.md]: Should we add a part suggesting to update remote link for local repos? Something like:

If you have a local clone of the rustc-dev-guide repository, we recommand that you update the remote URL to the GitHub repo:
```sh
git remote set-url origin https://github.com/rust-lang/rustc-dev-guide.git
```

**Comment by nikomatsakis** [comment]
@mark-i-m should this be merged now?

**Comment by mark-i-m** [comment]
@nikomatsakis Thanks :) Not yet, as we haven't actually done the rename yet, and I need to update it.

**Comment by nikomatsakis** [comment]
@mark-i-m now? :)

**Comment by mark-i-m** [comment]
Personally, I think so, but I think @spastorino wanted to wait until we had finished fixing toolstate.

**Comment by pietroalbini** [comment]
I don't think we should block on that, it's just an implementation detail of the rustc repo.

**Comment by spastorino** [comment]
Yes, this could be merged now I think. I didn't say I wanted to wait for toolstate, I believe :).

**Pull request merged**